showcase/app/templates/layouts/app-frame/frameless/demo-full-app-frame-with-side-nav.hbs

```diff
@@ -81,7 +81,7 @@
     </Hds::SideNav>
   </Frame.Sidebar>
 
-  <Frame.Main id="main">
+  <Frame.Main>
```

Comment:
blocker: since we're now using the default, we'd need to update [the `Hds::SideNav` in this example](https://github.com/hashicorp/design-system/pull/2399/files#diff-7df38fbd368e19b3ccf924f166524785c962698500ea45a725ed68139d7186a1R13) to either point to `hds-main` (or to remove it and rely on the default) otherwise [the skip link will not work on this page](https://hds-showcase-git-hds-3672-a11y-refocus-bug-hashicorp.vercel.app/layouts/app-frame/frameless/demo-full-app-frame-with-side-nav)